### ISSUE TITLE
[RFC] How to review the same PR again?

### ISSUE THREAD

**Issue by chshersh** [kowainik/hintman#106]
Currently, Hintman reviews PRs when they are open. But sometimes it might be desired to rereview them again. It's possible to handle _`request_review`_ action for Hintman and review again. But the question, what parts should we review again?

**Comment by vrom911** [comment]
I would say that the whole diff from the master branch if it's possible to collapse at once in the API...

**Comment by chshersh** [comment]
Another question: should `hintman` be triggered on `Reopened` PRs? Or should we make it configurable with `disabled` by default to not annoy people?

**Comment by vrom911** [comment]
Maybe it only should be triggered on newly opened PRs or on explicit demand (this request the review button near the reviewer's icon), what do you think?

**Comment by chshersh** [comment]
@vrom911 Makes sense. Let's trigger only on opened PRs and on the explicit request. Unfortunately, with GitHub UI it's very easy to accidentally close and reopen PR.
